Adding dynamic playlist about jcstress from /jcstress/tests-custom/sr…

jabrena opened the pull request:
Playlist generated by the following JBang script:
https://github.com/jabrena/aqa-tests-experiments/blob/main/PlaylistGenerator.java

llxia commented:
Thanks @jabrena . Any idea how long these tests will take?

smlambert commented:
Tested on ubuntu-latest, using run-aqa, in the workflow file in jabrena's repo, see a run here:
https://github.com/jabrena/aqa-tests-experiments/actions/runs/3048691650/jobs/4914022165

I will kick off some runs on other platforms in Jenkins out of interest for gathering execution times.

llxia commented:
I noticed lots of the following errors. Not sure if the tests ran properly.
```
Unrecognized option: -Xt
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.



FATAL: No JVM configurations to run with.
```

smlambert commented:
I believe those errors are to be ignored (if they follow the N/A moniker).  It seems the jcstress framework will attempt run with various options, if it hits some errors where the option is not available, it deems that particular test to be not applicable (N/A).  

```
Initializing and probing the target VM:
 (all failures are non-fatal, but may affect testing accuracy)

----- [OK] Unlocking diagnostic VM options
----- [OK] Disabling performance data collection
----- [OK] Checking for C1 availability
----- [OK] Checking for C2 availability
----- [OK] Trimming down the VM heap size to 256M
----- [OK] Trimming down the number of parallel GC threads
----- [OK] Trimming down the number of concurrent GC threads
----- [OK] Trimming down the number of G1 concurrent refinement GC threads
----- [OK] Trimming down the number of compiler threads
----- [OK] Testing @Contended works on all results and infra objects
----- [OK] Unlocking debug information for non-safepoints
----- [OK] Unlocking C2 local code motion randomizer
----- [OK] Unlocking C2 global code motion randomizer
----- [N/A] Unlocking C2 iterative global value numbering randomizer
Unrecognized VM option 'StressIGVN'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Unlocking C2 conditional constant propagation randomizer
Unrecognized VM option 'StressCCP'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Checking if C2 randomizers accept stress seed
Unrecognized VM option 'StressSeed=42'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [OK] Testing allocation profiling
----- [OK] Testing Thread.onSpinWait
----- [OK] Testing PrintAssembly
----- [OK] Testing compiler directives

Probing what VM configurations are available:
 (failures are non-fatal, but may miss some interesting cases)

----- [N/A] [-t, BooleanFencedTest]
Unrecognized option: -Xt
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.



FATAL: No JVM configurations to run with.

```

If other errors are seen (not behind the N/A), then we should consider them as potential issues and we can engage the mailing lists at https://github.com/openjdk/jcstress#interpreting-the-results

jabrena commented:
Later, I will share some logs from a standalone execution with Maven and Gradle using the unique plugin in the market.
Yesterday, reviewed in a quick way the logs and everything is fine, but I will share some logs to be more transparent :)

llxia commented:
Could this be related to our setup? 

https://github.com/adoptium/aqa-tests/blob/effa6d5307c152e2db941fa430d8aeefadd64cc7/system/jcstress/playlist.xml#L29

It looks like `  -t IntFencedTest ` is passed into `-jvmArgs`. Is this correct? See the [console output](https://pipelines.actions.githubusercontent.com/serviceHosts/9e9ed4ae-38ae-4b8d-b333-2951770e625a/_apis/pipelines/1/runs/3/signedlogcontent/2?urlExpires=2022-09-14T15%3A10%3A02.8066130Z&urlSigningMethod=HMACV1&urlSignature=97udO9F323BKZHLCy7ERl28xDB7hxXfhjEnAyLpL08U%3D):
```
... -jar "/home/runner/work/aqa-tests-experiments/aqa-tests-experiments/aqa-tests/TKG/../TKG/lib/jcstress-latest.jar" -jvmArgs "  -t IntFencedTest"; 
```

llxia commented:
Thanks @jabrena . Do you see the similar error in the standalone execution with Maven and Gradle?

smlambert commented:
Yes, I believe you are right @llxia, that is not correct.  The original placeholder testcase that @Mesbah-Alam added also suffers from this.

If JVM_OPTIONS="NoOptions", then the -jvmArgs option should not be present.

We should handle this case with logic in a system.mk file.  Analogous to what we do for certain types of option handling in [openjdk.mk](https://github.com/adoptium/aqa-tests/blob/master/openjdk/openjdk.mk), [jck.mk](https://github.com/adoptium/aqa-tests/blob/master/jck/jck.mk), etc. 

Pseudocode to add to a system.mk file:

JCSTRESS_OPTIONS :=
ifdef JVM_OPTIONS
    JCSTRESS_OPTIONS += -jvmArgs $(JVM_OPTIONS)
endif

and change the playlist for these targets to be `<command>` tag to use JCSTRESS_OPTIONS as in:
`<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -jar $(Q)$(LIB_DIR)$(D)jcstress-latest.jar$(Q)  $(Q)$(JCSTRESS_OPTIONS)$(Q) -t SampleTestBench$(Q);  $(TEST_STATUS)</command>`

smlambert commented:
thanks @Mesbah-Alam - also see my note here about making it even more generic:

https://github.com/adoptium/aqa-tests/issues/3546#issuecomment-1247029156

jabrena commented:
Oki, with that changes, I will regenerate the playlist again. For my log insights, I will share tomorrow

jabrena commented:
Maven project running classic:

```
Java Concurrency Stress Tests
---------------------------------------------------------------------------------
Rev: null, built by jabrena with 11.0.16 at null

Burning up to figure out the exact CPU count....... done!

Probing the target OS:
 (all failures are non-fatal, but may affect testing accuracy)

----- [N/A] Trying to set global affinity with taskset
Cannot run program "taskset": error=2, No such file or directory
----- [N/A] Trying to set per-thread affinity with syscalls
Not implemented

Initializing and probing the target VM:
 (all failures are non-fatal, but may affect testing accuracy)

----- [OK] Unlocking diagnostic VM options
----- [OK] Disabling performance data collection
----- [OK] Checking for C1 availability
----- [OK] Checking for C2 availability
----- [OK] Trimming down the VM heap size to 256M
----- [OK] Trimming down the number of parallel GC threads
----- [OK] Trimming down the number of concurrent GC threads
----- [OK] Trimming down the number of G1 concurrent refinement GC threads
----- [OK] Trimming down the number of compiler threads
----- [OK] Testing @Contended works on all results and infra objects
----- [OK] Unlocking debug information for non-safepoints
----- [OK] Unlocking C2 local code motion randomizer
----- [OK] Unlocking C2 global code motion randomizer
----- [N/A] Unlocking C2 iterative global value numbering randomizer
Unrecognized VM option 'StressIGVN'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Unlocking C2 conditional constant propagation randomizer
Unrecognized VM option 'StressCCP'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Checking if C2 randomizers accept stress seed
Unrecognized VM option 'StressSeed=42'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [OK] Testing allocation profiling
----- [OK] Testing Thread.onSpinWait
----- [OK] Testing PrintAssembly
----- [OK] Testing compiler directives

Probing what VM configurations are available:
 (failures are non-fatal, but may miss some interesting cases)
 (split compilation is requested and compiler directives are available)

----- [OK] []
----- [OK] [-XX:+StressLCM, -XX:+StressGCM]

Detecting CPU topology and computing scheduling classes:
  Fallback topology, faking CPU layout, downgrading to "NONE" affinity mode

  Scheduling classes for matching tests:
    2 actors:
      (PG free, CG free), (PG free, CG free)

  Test configuration:
    Test preset mode: "default"
    Hardware CPUs in use: 8
    Spinning style: Thread.onSpinWait()
    Test selection: ".*"
    Forks per test: 1 normal, 5 stress
    Iterations per fork: 5
    Time per iteration: 100 ms
    Test stride: 40 strides x 256 tests, but taking no more than 128 Mb
    Test result blob: "jcstress-results-2022-09-15-21-19-26.bin.gz"
    Test results: "results/"

  Attached the non-interactive output stream.
  Printing the progress line at most every 15000 milliseconds.

(ETA: in 00:00:32; at Thu, 2022-09-15 21:20:05)
(Sampling Rate: 6.28 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 4 allocated)
(Results: 34 planned; 1 passed, 0 failed, 0 soft errs, 0 hard errs)

(ETA: now)
(Sampling Rate: 29.58 M/sec)
(JVMs: 0 starting, 0 running, 0 finishing)
(CPUs: 8 configured, 0 allocated)
(Results: 34 planned; 34 passed, 0 failed, 0 soft errs, 0 hard errs)



RUN RESULTS:
  Interesting tests: No matches.

  Failed tests: No matches.

  Error tests: No matches.

  All remaining tests: 1 matching test results. Use -v to print them.

  HTML report generated at /Users/jabrena/Documents/DATA/RESEARCH/2021/internet/java/aqa-tests-experiments/jcstress-maven-demo/target/reports/jcstress/results/index.html
```

jabrena commented:
Gradle demo:

```
jabrena@Juans-MacBook-Pro jcstress-gradle-demo % ./gradlew clean build jcstress

> Task :jcstress
Java Concurrency Stress Tests
---------------------------------------------------------------------------------
Rev: 5297e983aa2bc98d, built by shade with 11.0.12 at 2021-12-02T09:54:27Z

Burning up to figure out the exact CPU count....... done!

Probing the target OS:
 (all failures are non-fatal, but may affect testing accuracy)

----- [N/A] Trying to set global affinity with taskset
Cannot run program "taskset": error=2, No such file or directory
----- [N/A] Trying to set per-thread affinity with syscalls
Not implemented

Initializing and probing the target VM:
 (all failures are non-fatal, but may affect testing accuracy)

----- [OK] Unlocking diagnostic VM options
----- [OK] Disabling performance data collection
----- [OK] Checking for C1 availability
----- [OK] Checking for C2 availability
----- [OK] Trimming down the VM heap size to 256M
----- [OK] Trimming down the number of parallel GC threads
----- [OK] Trimming down the number of concurrent GC threads
----- [OK] Trimming down the number of G1 concurrent refinement GC threads
----- [OK] Trimming down the number of compiler threads
----- [OK] Testing @Contended works on all results and infra objects
----- [OK] Unlocking debug information for non-safepoints
----- [OK] Unlocking C2 local code motion randomizer
----- [OK] Unlocking C2 global code motion randomizer
----- [N/A] Unlocking C2 iterative global value numbering randomizer
Unrecognized VM option 'StressIGVN'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Unlocking C2 conditional constant propagation randomizer
Unrecognized VM option 'StressCCP'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Checking if C2 randomizers accept stress seed
Unrecognized VM option 'StressSeed=42'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [OK] Testing allocation profiling
----- [OK] Testing Thread.onSpinWait
----- [OK] Testing PrintAssembly
----- [OK] Testing compiler directives

Probing what VM configurations are available:
 (failures are non-fatal, but may miss some interesting cases)
 (split compilation is requested and compiler directives are available)

----- [OK] [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
----- [OK] [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]

Detecting CPU topology and computing scheduling classes:
  Fallback topology, faking CPU layout, downgrading to "NONE" affinity mode

  Scheduling classes for matching tests:
    2 actors:
      (PG free, CG free), (PG free, CG free)

  Test configuration:
    Test preset mode: "default"
    Hardware CPUs in use: 8
    Spinning style: Thread.yield()
    Test selection: ".*"
    Forks per test: 1 normal, 5 stress
    Iterations per fork: 5
    Time per iteration: 200 ms
    Test stride: 40 strides x 256 tests, but taking no more than 128 Mb
    Test result blob: "jcstress-results-2022-09-15-21-34-59.bin.gz"
    Test results: "/Users/jabrena/Documents/DATA/RESEARCH/2021/internet/java/aqa-tests-experiments/jcstress-gradle-demo/build/reports/jcstress/"

  Attached the non-interactive output stream.
  Printing the progress line at most every 15000 milliseconds.


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  13,826,048  100.00%  Acceptable  All acceptable

(ETA: in 00:00:48; at Thu, 2022-09-15 21:35:55)
(Sampling Rate: 8.86 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 1 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #3

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  14,481,408  100.00%  Acceptable  All acceptable

(ETA: in 00:00:24; at Thu, 2022-09-15 21:35:31)
(Sampling Rate: 17.56 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 2 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #4

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  12,843,008  100.00%  Acceptable  All acceptable

(ETA: in 00:00:16; at Thu, 2022-09-15 21:35:22)
(Sampling Rate: 25.40 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 3 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  5,459,968  100.00%  Acceptable  All acceptable

(ETA: in 00:00:11; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 28.72 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 4 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #4

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  16,990,208  100.00%  Acceptable  All acceptable

(ETA: in 00:00:16; at Thu, 2022-09-15 21:35:24)
(Sampling Rate: 21.68 M/sec)
(JVMs: 1 starting, 2 running, 0 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 5 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #5

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  5,685,248  100.00%  Acceptable  All acceptable

(ETA: in 00:00:13; at Thu, 2022-09-15 21:35:20)
(Sampling Rate: 23.52 M/sec)
(JVMs: 2 starting, 2 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 6 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #2

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  14,553,088  100.00%  Acceptable  All acceptable

(ETA: in 00:00:10; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 28.44 M/sec)
(JVMs: 2 starting, 2 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 7 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #5

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  5,941,248  100.00%  Acceptable  All acceptable

(ETA: in 00:00:09; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 30.21 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 8 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  7,067,648  100.00%  Acceptable  All acceptable

(ETA: in 00:00:11; at Thu, 2022-09-15 21:35:20)
(Sampling Rate: 22.09 M/sec)
(JVMs: 1 starting, 2 running, 2 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 9 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #2

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  7,886,848  100.00%  Acceptable  All acceptable

(ETA: in 00:00:10; at Thu, 2022-09-15 21:35:19)
(Sampling Rate: 23.79 M/sec)
(JVMs: 1 starting, 2 running, 1 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 10 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #1

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  6,258,688  100.00%  Acceptable  All acceptable

(ETA: in 00:00:08; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 25.13 M/sec)
(JVMs: 1 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 11 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #1

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  6,258,688  100.00%  Acceptable  All acceptable

(ETA: in 00:00:07; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 26.53 M/sec)
(JVMs: 1 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 12 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  17,133,568  100.00%  Acceptable  All acceptable

(ETA: in 00:00:08; at Thu, 2022-09-15 21:35:19)
(Sampling Rate: 23.17 M/sec)
(JVMs: 1 starting, 3 running, 1 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 13 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  13,651,968  100.00%  Acceptable  All acceptable

(ETA: in 00:00:07; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 25.49 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 14 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #5

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  15,546,368  100.00%  Acceptable  All acceptable

(ETA: in 00:00:06; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 28.07 M/sec)
(JVMs: 1 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 15 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #4

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  4,425,728  100.00%  Acceptable  All acceptable

(ETA: in 00:00:06; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 28.80 M/sec)
(JVMs: 1 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 16 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #3

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  19,591,168  100.00%  Acceptable  All acceptable

(ETA: in 00:00:06; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 26.11 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 17 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #5

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  13,539,328  100.00%  Acceptable  All acceptable

(ETA: in 00:00:06; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 27.90 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 18 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #2

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  14,020,608  100.00%  Acceptable  All acceptable

(ETA: in 00:00:05; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 29.71 M/sec)
(JVMs: 1 starting, 3 running, 1 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 19 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #4

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  4,927,488  100.00%  Acceptable  All acceptable

(ETA: in 00:00:04; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 30.31 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 20 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #4

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  16,570,368  100.00%  Acceptable  All acceptable

(ETA: in 00:00:04; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 27.52 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 21 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #2

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  15,710,208  100.00%  Acceptable  All acceptable

(ETA: in 00:00:04; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 29.28 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 22 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  17,584,128  100.00%  Acceptable  All acceptable

(ETA: in 00:00:03; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 31.19 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 23 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  5,408,768  100.00%  Acceptable  All acceptable

(ETA: in 00:00:03; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 31.73 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 24 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #3

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  14,675,968  100.00%  Acceptable  All acceptable

(ETA: in 00:00:03; at Thu, 2022-09-15 21:35:18)
(Sampling Rate: 28.89 M/sec)
(JVMs: 1 starting, 2 running, 1 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 25 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  6,330,368  100.00%  Acceptable  All acceptable

(ETA: in 00:00:02; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 29.50 M/sec)
(JVMs: 1 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 26 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  19,294,208  100.00%  Acceptable  All acceptable

(ETA: in 00:00:02; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 31.42 M/sec)
(JVMs: 1 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 27 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #2

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  5,879,808  100.00%  Acceptable  All acceptable

(ETA: in 00:00:01; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 31.85 M/sec)
(JVMs: 0 starting, 4 running, 0 finishing)
(CPUs: 8 configured, 8 allocated)
(Results: 34 planned; 28 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #5

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  15,822,848  100.00%  Acceptable  All acceptable

(ETA: in 00:00:01; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 29.51 M/sec)
(JVMs: 1 starting, 2 running, 2 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 29 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #3

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  6,596,608  100.00%  Acceptable  All acceptable

(ETA: in 00:00:01; at Thu, 2022-09-15 21:35:17)
(Sampling Rate: 30.06 M/sec)
(JVMs: 0 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 30 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C2
    actor2: C1

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  15,054,848  100.00%  Acceptable  All acceptable

(ETA: in 00:00:00; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 31.37 M/sec)
(JVMs: 0 starting, 3 running, 0 finishing)
(CPUs: 8 configured, 6 allocated)
(Results: 34 planned; 31 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: Interpreter
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant, -XX:+StressLCM, -XX:+StressGCM]
  Fork: #3

      RESULT    SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  6,483,968  100.00%  Acceptable  All acceptable

(ETA: in 00:00:00; at Thu, 2022-09-15 21:35:16)
(Sampling Rate: 31.70 M/sec)
(JVMs: 0 starting, 2 running, 0 finishing)
(CPUs: 8 configured, 4 allocated)
(Results: 34 planned; 32 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: Interpreter

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  16,580,608  100.00%  Acceptable  All acceptable

(ETA: now)
(Sampling Rate: 29.96 M/sec)
(JVMs: 0 starting, 0 running, 1 finishing)
(CPUs: 8 configured, 0 allocated)
(Results: 34 planned; 33 passed, 0 failed, 0 soft errs, 0 hard errs)


.......... [OK] org.openjdk.jcstress.SampleTest

  Scheduling class:
    actor1: package group free, core group free
    actor2: package group free, core group free

  CPU allocation: unspecified

  Compilation: split
    actor1: C1
    actor2: C2

  JVM args: [-XX:+UnlockDiagnosticVMOptions, -XX:+WhiteBoxAPI, -XX:-RestrictContended, -Dfile.encoding=UTF-8, -Duser.country=ES, -Duser.language=en, -Duser.variant]
  Fork: #1

      RESULT     SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  43,122,688  100.00%  Acceptable  All acceptable

(ETA: now)
(Sampling Rate: 33.31 M/sec)
(JVMs: 0 starting, 0 running, 0 finishing)
(CPUs: 8 configured, 0 allocated)
(Results: 34 planned; 34 passed, 0 failed, 0 soft errs, 0 hard errs)

(ETA: now)
(Sampling Rate: 30.87 M/sec)
(JVMs: 0 starting, 0 running, 0 finishing)
(CPUs: 8 configured, 0 allocated)
(Results: 34 planned; 34 passed, 0 failed, 0 soft errs, 0 hard errs)



RUN RESULTS:
  Interesting tests: No matches.

  Failed tests: No matches.

  Error tests: No matches.

  All remaining tests: 1 matching test results.

.......... [OK] org.openjdk.jcstress.SampleTest

  Results across all configurations:

      RESULT      SAMPLES     FREQ      EXPECT  DESCRIPTION
  0, 0, 0, 0  425,203,712  100.00%  Acceptable  All acceptable



  HTML report generated at /Users/jabrena/Documents/DATA/RESEARCH/2021/internet/java/aqa-tests-experiments/jcstress-gradle-demo/build/reports/jcstress/index.html
```

jabrena commented:
In both examples, I am using the classic example: `SampleTestBench`
from:  https://github.com/openjdk/jcstress/blob/master/jcstress-benchmarks/src/main/java/org/openjdk/jcstress/SampleTestBench.java

jabrena commented:
@llxia @smlambert, if you observe in the logs, JCStress is very verbose you can see in valid executions this kind of errors:

```
----- [N/A] Unlocking C2 conditional constant propagation randomizer
Unrecognized VM option 'StressCCP'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.

----- [N/A] Checking if C2 randomizers accept stress seed
Unrecognized VM option 'StressSeed=42'
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

llxia commented:
Thanks @jabrena .
PR https://github.com/adoptium/aqa-tests/pull/3968 is merged to fix the `-Xt` error. Please rebase and update the playlist in this PR/

jabrena commented:
Apologies for the delay, this weekend I will do this small change in my branch.
